gatsby-node.js

```diff
@@ -499,6 +504,12 @@ exports.createPages = async function ({ actions, graphql }) {
     }
 
     if (person.path.includes('alumni')) {
+      var originalPath = getLastSegment(person.path); // Gets the path, removing the /alumni/ prefix
+      actions.createRedirect({
```

Comment:
```suggestion
      const originalPath = getLastSegment(person.path); // Gets the path, removing the /alumni/ prefix
      actions.createRedirect({
```
I wouldn't use var. Just want to avoid hoisting https://www.w3schools.com/js/js_hoisting.asp

Reply:
Actioned in #687